package.json

```diff
@@ -8,7 +8,9 @@
   "dependencies": {
     "css-layout": "^1.1.0",
     "d3": "^3.5.4",
+    "d3-path": "^0.1.5",
```

Comment:
Do we need to depend on `d3-path`, isn't the transitive dependency via `d3fc-path` sufficient?